joomla/joomla-cms
# Changing loading order between Redirect and Logout system plugins at install time

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/19486

### Summary of Changes
Changing the order of these plugins in joomla.sql for the Redirect plugin to load first by default.

### Testing Instructions
Make a Joomla clean install including this PR.
Check the plugins Manager, filter by system plugins.
Now the Redirect plugin is displayed before the Logout plugin when filtering by Type Ascending or Ordering ascending.
Enable the Redirect plugin.
Test the issue described https://github.com/joomla/joomla-cms/pull/19233 i.e. logout via the Login Form Module while editing an article.

### Expected result
No 403, but a clean redirect to the Home page
